### PR TITLE
Improved publisher onboarding

### PR DESCRIPTION
**Pull request by davidfischer**
Add a few helpful notes for onboarding publishers

## Screenshots
Screenshot from 2022-04-26 14-45-15

(This screenshot is simulated because a publisher with views won't get this section)

### CODE REVIEW

File: adserver/templates/adserver/publisher/overview.html [hunk 1]
```diff
@@ -74,11 +74,34 @@ <h6 class="text-muted text-center">{% trans 'Clicks' %}</h6>
...
+      <div class="row">
+        <div class="col">
+          <h3>{% trans 'There are three steps to getting approved for paid ads and to start receiving payouts.' %}</h3>
+          <ol class="mt-5 lead">
+            <li>
+              {% url "publisher_embed" publisher.slug as embed_url %}
+              <p>{% blocktrans %}Setup your ads with our <a href="{{ embed_url }}">ad client embed</a>. You are already be approved for unpaid ads.{% endblocktrans %}</p>
+            </li>
+            <li>
+              <p>{% blocktrans %}After setting up unpaid ads on your site, ensure that traffic is coming through to this dashboard.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/publisher/overview.html:88]:
```suggestion
              <p>{% blocktrans %}After setting up unpaid ads on your site, ensure that views are being counted in this dashboard.{% endblocktrans %}</p>
```

File: adserver/templates/adserver/publisher/overview.html [hunk 2]
```diff
@@ -74,11 +74,34 @@ <h6 class="text-muted text-center">{% trans 'Clicks' %}</h6>
...
+            </li>
+            <li>
+              {% url "support" as support_url %}
+              {% url "publisher_settings" publisher.slug as publisher_settings_url %}
+              <p>{% blocktrans %}<a href="{{ support_url }}?subject=Paid+ads+approval&body=I+would+like+to+be+approved+for+paid+ads+on+my+site.">Request approval</a> for paid ads. You will also need to configure your payout preferences in your <a href="{{ publisher_settings_url }}">settings</a>.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/publisher/overview.html:93]: I'd really prefer people to reply to the email thread they started originally, but there isn't a way to link to that easily. This is a reasonable substitute. 👍

**Review by ericholscher — APPROVED** [review]
Looks great 👍

File: adserver/templates/adserver/publisher/overview.html (hunk 2 above)
Comment (davidfischer) [reply adserver/templates/adserver/publisher/overview.html:93]: Same

**Pull request merged**